test/gtest/ucp/test_ucp_memheap.cc

```diff
@@ -170,6 +173,10 @@ void test_ucp_memheap::test_blocking_xfer(blocking_send_func_t send,
         goto out;
     }
 
+    if (!malloc_allocate) {
```

Comment:
can set it in the check above at line 162 (though flags needs to be initialized before that check)